lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -207,6 +207,7 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
 
     it "STAKE_POOLS_JOIN_01rewards - \
         \Can join a pool, earn rewards and collect them" $ \ctx -> runResourceT $ do
+        liftIO $ flakyBecauseOf "#2415"
```

Comment:
I believe this is the only e2e test in the suite that is doing a successful rewards withdrawal so it would be a pity not to run it at all. If we deem it is permanently flaky than maybe it'd be better to loosen some assertions rather than turning it off?

For instance, not check that the rewards are zero after withdrawal, but that the rewards are lower?

Reply:
Point noted, thanks @piotr-iohk.